### DISCUSSION TITLE
Migrating from Forestry to Tina on Gatsby

### DISCUSSION THREAD

**Discussion by Kosiso29** [tinacms/tinacms#3823] [Experimental]
Hello, Please I just installed the tina cli in the terminal for my existing Gatsby (Forestry) project and after the installation, I ran the command `yarn tinacms dev -c "gatsby develop"` to run the application in dev mode, but it keeps ending in `Done` without throwing any errors or creating any dev servers.
Here's a sample of what I get when I run it.
<img width="583" alt="Screenshot 2023-04-18 at 18 11 39" src="https://user-images.githubusercontent.com/65488353/232853310-47e84bf5-0eb3-4dd3-b9e6-ed0e270e80e2.png">

Kindly help with a feedback on how to deal with this, cos I kinda have a system that seems to be fine when it's not. I need a way to run my dev server and see my working application.

**Comment by kslstn** [comment]
I'm not a Gatsby user, but I think you need to include 'npm' in the subprocess parameter. Something like: `npm tinacms dev -c "npm gatsby develop -o"`. Whatever the command is you use to run a normal dev server should be passed on to Tina.

**Reply by Kosiso29** [reply]
Thank you very much @kslstn it's worked. I used `npm tinacms dev -c "yarn gatsby develop -o"`
